### PR TITLE
feat-upvotes-integration

### PR DESCRIPTION
**Pull request by purplena**
I do not understand how I can make a sort of a toggle to use the same method in the controller (maybe it is not event possible) to make upvotes and downvotes in the same time.

### CODE REVIEW

File: database/seeders/UpvoteSeeder.php [hunk 1]
```diff
@@ -14,10 +14,10 @@ public function run(): void
     $requestedSongs = RequestedSong::all();
     $users = User::where('role', 2)->get();
```
Comment (stephdotnet) [inline database/seeders/UpvoteSeeder.php:15]: $users = User::where('role', User::ROLE_ADMIN)->get();

**Review by stephdotnet — APPROVED** [review]
QQs commentaires que tu peux fix sur ton autre PR

File: app/Http/Controllers/RequestedSongController.php [hunk 2]
```diff
@@ -24,28 +24,22 @@ public function __construct(protected SpotifyApi $spotifyApi)
...
     return RequestedSongResource::collection($requestedSongs);
   }
 
+
+
   public function search(Request $request): JsonResponse
   {
     // $results = app(SpotifyApi::class)->getClient()->search($request->query('q'), 'track');
     $results = $this->spotifyApi->getClient()->search($request->query('q'), 'track');
+    dd(collect($results->tracks->items));
```
[the remark this answers is not in the document]
Comment (purplena) [reply app/Http/Controllers/RequestedSongController.php:43]: I have already deleted.

File: database/seeders/UpvoteSeeder.php (hunk 1 above)
Comment (purplena) [reply database/seeders/UpvoteSeeder.php:15]: Wrote `$users = User::where('role', User::ROLE_CLIENT)->get();` because `2 = ROLE_CLIENT`

**Pull request merged**